onnxruntime/core/framework/session_options.h

```diff
@@ -113,23 +114,11 @@ struct SessionOptions {
   // To add an configuration to this session, call OrtApis::AddSessionConfigEntry
   // The configuration keys and value formats are defined in
   // /include/onnxruntime/core/session/onnxruntime_session_options_config_keys.h
-  std::unordered_map<std::string, std::string> session_configurations;
+  ConfigOptions session_configurations;
   std::unordered_map<std::string, const OrtValue*> initializers_to_share_map;
```

Comment:
can we maybe call this member config_options? the 'session' part is implied by them coming from SessionOptions

Reply:
Alternatively, could make the ConfigOptions member private and keep the GetConfigOrDefault as a pass through call to it.

Reply:
called this member config_options